ocs_ci/ocs/utils.py

```diff
@@ -1190,6 +1190,9 @@ def _collect_ocs_logs(
             ocs_must_gather_image_and_tag = mirror_image(
                 ocs_must_gather_image_and_tag, cluster_config
             )
+        if cluster_config.ENV_DATA.get("cluster_type") == constants.HCI_CLIENT:
```

Comment:
This will need to be changed in future PR to not include HCI string but it can be done as part of different PR dedicated to the name change.

Reply:
Yes, correct.